src/app/containers/Metadata/index.jsx

```diff
@@ -105,6 +108,8 @@ const MetadataContainer = ({
   const metaImage = image || defaultImage;
   const metaImageAltText = imageAltText || defaultImageAltText;
 
+  const { enabled: iTunesAppEnabled } = useToggle('smart_app_banner');
```

Comment:
What happens here when the toggle service is down? Could we get an error destructuring this object or would `iTunesAppEnabled` default to `undefined`, fail the 'truthy' check and just not add the meta tag?

Probably worth adding a test for this

Reply:
This is temporary - to allow us to test on the test environment, without releasing it to live. It will be removed very soon. 

As we saw earlier this week, when the toggles service went down, features controlled by toggles were hidden, because the enabled value was `null` and therefore failed truthy checks. https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/getToggles/index.js#L35